source/content/debug-cache.md

```diff
@@ -48,53 +48,52 @@ Generated by WordPress or Drupal to send data back to the browser. The platform
 x-styx-req-id: 7f93c166-53fe-11ea-803e-b26d7703e33f
-date: Thu, 20 Feb 2020 16:31:51 GMT
-x-served-by: cache-mdw17379-MDW, cache-chi21146-CHI
-x-cache: MISS, MISS
-x-cache-hits: 0, 0
-x-timer: S1582216311.492451,VS0,VE204
-vary: Accept-Encoding, Cookie, Cookie
+date: Tue, 08 Dec 2020 16:41:37 GMT
+x-served-by: cache-mdw17348-MDW
+x-cache: MISS
+x-cache-hits: 0
+x-timer: S1607445693.448541,VS0,VE4134
+vary: Accept-Encoding, Cookie
+x-robots-tag: noindex
 age: 0 // highlight-line
 accept-ranges: bytes
 via: 1.1 varnish
+content-length: 41278
 ```
 
-If you see `Age: 0` after multiple requests, your site is not caching properly.
+This specific example is from a Drupal site that has anonymous caching enabled, but has "none" set for expiration of cached pages within Drupal's performance settings (`/admin/config/development/performance`). For details, see [Drupal Performance and Caching Settings](/drupal-cache).
 
-This particular example is covered in [WordPress Plugins and Themes with Known Issues](/plugins-known-issues#gdpr-cookie-consent).
+### Set-Cookie Headers
+Pantheon's platform will not cache a response that contains the `set-cookie` header.
 
-### Cookie Name Prefix
+<Alert title="Note" type="info" >
 
-Pantheon's platform will not cache a response that contains the `set-cookie` header.
+Do not use Curl to debug this scenario. Instead, investigate `set-cookie` issues and fixes from your browser, for instructions see [Test Global CDN with Browser Headers](/test-global-cdn-caching#test-global-cdn-with-browser-headers). Curl does not handle cookie data in the way that your browser does and effectively misrepresents cookie and caching behavior as a result.
 
-The best way to utilize cookies on Pantheon is by having the cookie name match the `STYXKEY[a-zA-Z0-9_-]+` naming convention, and loading them in the first load, not on every page load. Refer to the sample code outlined in [Working with Cookies on Pantheon](/cookies#cache-varying-cookies).
+</Alert>
 
-For example, if a theme file sets a cookie as:
+Search for instances across the site's codebase and once isolated, edit the cookie name so that is matches the [`STYXKEY[a-zA-Z0-9_-]+` naming convention](/caching-advanced-topics#using-styxkey).
 
-```php
-$cookie_name = 'cookies_disclaimer_id_'.$cookie_page_id;
-```
+Next, wrap a conditional statement around the `setcookie();` function to check whether the cookie is already defined in the request. The goal of this effort is to send `set-cookie` only on the initial request, and never on subsequent requests. For code example, see [Working with Cookies on Pantheon](/cookies#cache-varying-cookies).
 
-Edit to:
+The initial request will still break through cache - that's expected, because it would not find anything defined for the cookie. Subsequent requests back to the site should find the cookie data defined in the request headers. Subsequent requests should not be sending `set-cookie` and should now be served by cache.
```

Comment:
In the last sentence, should we change "requests" to "returns"? If not, can we merge it with the previous sentence, as it seems to say the same thing?

Reply:
Yes you can merge